logforward.tf

```diff
@@ -22,12 +22,18 @@ resource "datadog_integration_aws_lambda_arn" "datadog_forwarder" {
   depends_on = [aws_cloudformation_stack.datadog_forwarder]
```

Comment:
So, the statement below depends_on the new wait_datadog_forwarder timer. 

Outside of screaming "but why", does it make sense to have these two resources depend on two different things when they previously depended on the same thing? 

If yes, please ignore.

Reply:
I believe it is appropriate. I'm spacing it out because `datadog_integration_aws_lambda_arn` would routinely fail to create on firstrun. Fairly certain it's a bug in the Datadog provider, but I just couldn't be bothered to go _there_